### ISSUE TITLE
Recent issues with master branch

### ISSUE THREAD

**Issue by thibaultcha** [pcmanus/ccm#558]
I've been using `2.3.2` locally for a while, but my Travis-CI script was simply cloning this repo's `master` branch so always getting the latest code. Today, all of my builds started failing with errors such as:
```
/home/travis/.local/lib/python2.7/site-packages/ccm-2.4.4-py2.7.egg/ccmlib/node.py:509: UserWarning: Binary interface 127.0.0.1:9042 is not listening after 10 seconds, node may have failed to start.
```

This is my last successful build 3 days ago: https://travis-ci.org/thibaultcha/lua-cassandra/builds/164163446

It would appear that in the last 3 days, something got broken? The only commit I see in this lapse of time is 833f7894ecb4cfc0afceaa07342e85fcbd003e35, which, although I did not investigate the issue, appears like a good candidate.

**Comment by ptnapoleon** [comment]
That commit was merged today. I don't think anything was merged ~three days ago. Can you see if that node actually failed to start, or if the warning is being incorrectly triggered?

**Comment by thibaultcha** [comment]
Indeed, nothing for 3 days (but also, no builds on my side), and 3 days ago, the last build worked fine. But as of today (which includes this new commit), the builds stopped working with this error all over the place.

I will investigate the warning.

**Comment by thibaultcha** [comment]
PS: temporarily reverting to 18a50dfde2069e64a3124e2300ad1f0c1e08f908 fixed my builds: https://github.com/thibaultcha/lua-cassandra/blob/master/.ci/setup_env.sh#L22

**Comment by ptnapoleon** [comment]
That is interesting
